recipes/noiseflow/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set org = "shakeflow" %}
+{% set name = "noiseflow" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ org }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ae0f060010667a924f7281976762d85cf5ba2402ccb404a0bab7a5fc893188fe
+
+build:
+  number: 0
+  skip: True  # [python_impl == 'pypy']
+  
+
+outputs:
```

Comment:
I don't think you need `outputs` for this. There are several differences in behaviour that are very subtle and might be causing issues with your current setup. I'd try using a regular global level for this recipe and see if it fixes the Linux issues.

Reply:
Thanks, I will delete `outputs` and try again. Actually, I want to add some other output in the future, so I just add the `outputs` here.